python/paddle/fluid/layer_helper.py

```diff
@@ -147,8 +152,13 @@ def append_activation(self, input_var):
 
         if 'use_cudnn' in self.kwargs and self.kwargs.get('use_cudnn'):
             act['use_cudnn'] = self.kwargs.get('use_cudnn')
-        if 'use_mkldnn' in self.kwargs:
-            act['use_mkldnn'] = self.kwargs.get('use_mkldnn')
+
+        # priority order: use_mkldnn in kwargs then global
+        use_mkldnn = self.kwargs.get(
+            'use_mkldnn', core.globals().get("FLAGS_use_mkldnn", False))
+        if use_mkldnn:
+            act['use_mkldnn'] = use_mkldnn
+
```

Comment:
Maybe it is better to implement this functionality in C++ side. There are 2 reasons,

1. `FLAGS_use_mkldnn ` is a GFLAG, which is declared and better used in c++. 
2. There may be other way to add an activation op, for example, `_append_activation_in_dygraph`.

Reply:
I've changed tracer on the c++ side, so there is no need for the above changes anymore.